addons/l10n_uy/data/res_partner_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+
+    <record id="partner_dgi" model="res.partner">
+        <field name="name">DGI</field>
+        <field name="is_company" eval="True"/>
+        <field name='l10n_latam_identification_type_id' ref='it_rut'/>
+        <field name='country_id' ref='base.uy'/>
+    </record>
+
+    <record model='res.partner' id='partner_cfu'>
+        <field name='name'>Consumidor Final Anónimo</field>
+        <field name='l10n_latam_identification_type_id' ref='it_dni'/>
```

Comment:
At the current module status is not required at all.
Not sure if it is required to be defined for EDI. I thnik we are requiring in some place the partner should always have a identification type define of type UY

Reply:
@jco-odoo if there is no technical restriction on having the identification type,  I suggest we keep it, as it can save some time when testing worflows on EDI where this field is requiered, and doesn't harm to have set in this base module.

Reply:
Ok, for the moment we leave the identification type.

Reply:
if you have any comment @jco-odoo please let us know